gax-java/gax/src/main/java/com/google/api/gax/rpc/ServerStreamingCallSettings.java

```diff
@@ -272,6 +290,19 @@ public Builder<RequestT, ResponseT> setIdleTimeout(@Nonnull Duration idleTimeout
       return this;
     }
 
+    @Nonnull
+    public Duration getWaitTimeout() {
+      return waitTimeout;
+    }
+
+    /**
+     * See the class documentation of {@link ServerStreamingCallSettings} for a description of what
+     * the {@link #waitTimeout} does. {@link Duration#ZERO} disables the watchdog.
+     */
+    public void setWaitTimeout(@Nonnull Duration waitTimeout) {
```

Comment:
How do we plan to use it? Are we always expecting this field to be set by customers? Introducing a `Nonnull` public field to an existing setting is a breaking change because customers would have to update their code. We should have a default value for these timeout.

Reply:
Builder() initialize it with 0 (line 174) and we don't always expect user to set it.

Reply:
> Builder() initialize it with 0 (line 174) and we don't always expect user to set it.

Thanks, what does `Duration.ZERO` mean in this case? We wait indefinitely in between messages for ServerSreaming RPCs? Do we still expect customers to set it sometimes?

Reply:
0 means disable watchdog. This default is override in the client library https://github.com/googleapis/java-bigtable/blob/main/google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/stub/EnhancedBigtableStubSettings.java#LL641C50-L641C50 

we expect customer to set waitTimeout if they expect longer wait time between responses.

Reply:
Make sense, I updated the javadoc